Feature lidar setup

JanezCim opened the pull request:
Added LS lidar instructions to learn pages. This is a draft pull request because more discussion is yet to be done:

Unresolved questions:

- Since we are likely to add the LD lidar, we need to make sure we have the tutorials prepared for that. Now the biggest issue is that i still dont know how we are going to be calling this lidar in the tutorials. Internally we define them as LS and LD lidars, but since we want to be branding the LS lidar as our own, what the heck do we call it in the tutorials? I've refered to it as magni-default lidar or ubiquity robotics lidar? 
- The same question for little LS lidar - are we branding it as UR lidar? If yes, how do we call it.

Please rewiew the tutorials and give me feedback. Thanks!

MoffKalast commented on doing_more/ls_lidar_setup_kinetic.md:
```diff
@@ -0,0 +1,43 @@
+
+#### &uarr;[top]( https://ubiquityrobotics.github.io/learn/)
+
+Ubuntu 18.04
```
16.04?

MoffKalast reviewed (changes requested):
Found a few issues to resolve, also we should probably discuss the best way to launch lidars on our images in a more seamless fashion.

JanezCim commented on doing_more/ls_lidar_setup_common.md:
```diff
@@ -0,0 +1,85 @@
+
+#### &uarr;[top]( https://ubiquityrobotics.github.io/learn/)
+
+In this document it is described how to set up the LS301 lidar on the Magni.
```
which.... is also not bad :D

yes, will fix this

JanezCim commented on doing_more/ls_lidar_setup_common.md:
```diff
@@ -0,0 +1,85 @@
+title:  "Setting up magni-default lidar"
+permalink: default_lidar_setup
+---
+# Setting up ubiquity robotics default lidar
```
Ok, true, do you have any suggestion as to what to call it? Im fine with LSN301 and LD06 but we all need to agree on calling it that then. See description of PR for more comments about naming?

MoffKalast replied:
Right I missed that part of the description, I suppose LS-N301 and LD06 is good. Full names would be LeiShen N301 and LDRobot LD06.

JanezCim commented on doing_more/ls_lidar_setup_common.md:
```diff
@@ -0,0 +1,85 @@
+    cd ~/catkin_ws/
+    source devel/setup.bash
+    roslaunch lslidar_n301_decoder lslidar_n301_config.launch device_IP:=192.168.42.222
+    
```
yeah if those lidars will become default on magni, maybe the launches could be in magni repo as well.

JanezCim commented on doing_more/ls_lidar_setup_common.md:
```diff
@@ -0,0 +1,85 @@
+
+A static interface needs to be set on RPIs lan port by editing the interfaces file
+
+    sudo nano /etc/network/interfaces
```
Ah yes, im now finding ways to make this work also on focal, Untill then i've just added a todo in a comment

MoffKalast commented:
> You mean that on the magni_robot repo, noetic branch? Any idea how to make it more seamless? Id be happy to hear it. :)

Yeah I think it may be worthwhile to have it as a yaml param much like sonars_installed, except lidar_type or something similar. Much like it's done [in ezmap kinetic](https://github.com/UbiquityRobotics/ezmap/blob/8a0bd219d84acf260377a768f7384a94147ea0a1/ezmap_bringup/launch/core.launch#L101) right now for the time being.

JanezCim commented:
@MoffKalast @mjstn i've now renamed the ls lidar into ur50 lidar in all the tutorials. i've also implemented @MoffKalast s suggestions. I think its now ready to be reviewed. I think we think about adding tutorials for other lidars in other PRs

JanezCim commented:
@MoffKalast and @mjstn this has been stale for some time now but its ready to be reviewed and merged if all is alright.

MoffKalast reviewed (approved):
Ok yeah the changes seem good for now, looks alright to me. There will always be a few more issues found once it's hosted, so we can everything that's noted later.
